src/circuit/circuit_frame.rs

```diff
@@ -4641,74 +4632,132 @@ pub fn comparison_helper<F: LurkField, CS: ConstraintSystem<F>>(
 }
 
 // Enforce 0 <= num < 2ˆn.
+#[allow(dead_code)]
 pub fn enforce_at_most_n_bits<F: LurkField, CS: ConstraintSystem<F>>(
```

Comment:
Do we need to keep this?

Reply:
Removed, but kept popcount, should I remove popcount too? It may be useful in the future.

Reply:
Let's keep it.